README.md

```diff
@@ -8,46 +8,26 @@ versions of Julia.
 Certain features, including global runs, are not currently available on
 Windows due to limitations with our regridding software.
 
-## Models
-```
-Model <: AbstractModel
-```
-
-All models have the same abstract supertype, and have shared functionality.
-There is the the option to define new methods particular to your model, or to
-fall back on defaults. The functions each new Model type can define are:
-- make_compute_exp_tendency()
-- make_compute_imp_tendency()
-- make_update_aux()
-- make_exp_tendency()
-- make_imp_tendency()
-- initialize_prognostic()
-- initialize_auxiliary()
-- initialize()
-- prognostic_vars()
-- auxiliary_vars()
-
-Each model will also have some notion of a domain with coordinates, parameter sets,
-and boundary conditions or other prescribed drivers.
+Please note that the model used in the following works is not currently incorporated into
```

Comment:
@Yujie-W do we have all the correct references here? Let me know if you'd like to rephrase this.